crates/libcruby-sys/src/lib.rs

```diff
@@ -92,4 +92,7 @@ extern "C" {
                       arg: void_ptr,
                       state: *mut RubyException)
                       -> VALUE;
+
+    pub fn rb_str_new_cstr(string: c_string) -> VALUE;
+    pub fn rb_utf8_str_new(string: c_string, len: libc::c_long) -> VALUE;
```

Comment:
Looks like the `rb_utf8_str_new` is actually a C macro that delegates to either `rb_utf8_str_new_static` or `rb_utf8_str_new`. Do you happen to know what's the difference between the two and if that distinction matters for us? If so, it's probably better to re-export the macro as a C function in [helix_runtime](https://github.com/rustbridge/helix/blob/master/ruby/ext/helix_runtime/native/helix_runtime.c) and call that here.